community-membership.md

```diff
@@ -219,3 +219,33 @@ The following apply to the subproject for which one would be an owner.
 - Ensure a healthy process for discussion and decision making is in place.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
+
+### Resolving technical conflicts within a SIG
+
+From time to time, the Maintainers for a given OpenTelemetry SIG may be unable
+to reach consensus on a technical issue. While it's healthy and appropriate to
+make a sincere attempt to understand all points of view and consider the
+tradeoffs, it's also healthy to occasionally "disagree and commit."
+
+Within OpenTelemetry, SIG Maintainers are chosen specifically _because_ they
+are domain experts, so we would like to keep as much of the decision-making
+authority with the SIG Maintainers rather than immediately "escalating" to the
+overall OpenTelemetry-wide Technical Committee. As such, this is
+OpenTelemetry's process for resolving technical issues where Maintainers cannot
+reach consensus:
+
+1. The SIG Maintainers should succintly document the options under
+consideration as a GitHub issue within the SIG's respective repo (note that it
+is *not* required to document the complete framing and pros/cons – just the
+actual go-forward options themselves).
+2. Each SIG Maintainer must formally vote for their choice by commenting on
+that issue.
+3. The option receiving the most votes wins.
+4. If the vote is a tie, the OpenTelemetry TC should be brought into the
```

Comment:
If it reached the point when maintainers cannot come to a consensus the TCs involvement may be necessary even when the voting is not a tie. If a minority of maintainers feel so strongly opposed to idea that they are willing to escalate I think we should allow that escalation to happen. I do not feel that saying "the majority of maintainers always win" is the right approach.

So, as an alternate to this PR's proposal how about we simply say: if maintainers cannot reach a consensus within the SIG they can submit a formal arbitration request as outlined in items 1-2 above and the TC will be responsible to make decision. No voting is necessary and we only resort to a formal process when it is clear the regular process in the SIG is not working.

Again, I am not opposed to the proposal in this PR, but I wonder if a simpler alternate may be more attractive.

Reply:
@tigrannajaryan FYI, this PR is in response to an actual and outstanding technical deadlock happening in a SIG where there is an explicit desire for "an official process."

I share your desire for a simple approach but thought that filing a (short) issue that only outlines the options under consideration would be less complexity – and, frankly, drama – than escalating to the TC, agreeing on the framing document, etc.

Anyway, this is also just my two cents. I could also add something in here about how "the TC is always glad to hear both sides of a technical dispute and attempt to arbitrate or help find consensus"... thoughts?

Reply:
The issue is that escalating to TC could mean moving the decision from people on the SIG who know the language best to people on TC who may not know the language at all, which could be even less desirable outcome than a majority vote.

I do agree though that an attempt at mediation by TC could come before calling for a formal SIG vote. In such case TC can potentially bring a different, broader perspective.